src/main/java/com/kjetland/dropwizard/activemq/BrowserCallback.java

```diff
@@ -0,0 +1,10 @@
+package com.kjetland.dropwizard.activemq;
+
+import javax.jms.JMSException;
+import javax.jms.QueueBrowser;
+
+@FunctionalInterface
+public interface BrowserCallback<T> {
+
+    T withBrowser(QueueBrowser browser) throws JMSException;
```

Comment:
Maybe it's just me, but I think it would be easier to understand what's happening if the withBrowser method would be renamed to something that has a verb in it, like run, execute, 
The **with** prefix makes me think of a builder that still needs to be triggered to do something